bot/cogs/bot.py

```diff
@@ -60,6 +62,20 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
```

Comment:
Is having this directly in bot.py a good idea? It might be better to have it elsewhere and be imported, for readability. ( And have that docstring moved there )
But thats open to discussion.

Reply:
I think it's fine for now.